src/main/java/chess/domain/strategy/speical/RookMoveStrategy.java

```diff
@@ -0,0 +1,14 @@
+package chess.domain.strategy.speical;
+
+import chess.domain.position.Direction;
+import java.util.List;
+
+public class RookMoveStrategy extends SpecialPieceMoveStrategy{
+
+    private static final int DEFAULT_MAX_MOVE_DISTANCE = 7;
+    private static final List<Direction> DIRECTIONS = List.of(Direction.N, Direction.W, Direction.S, Direction.E);
```

Comment:
```suggestion
    private static final List<Direction> DIRECTIONS = Direction.straightDirections();
```

이런식으로 표현해볼 수도 있다고 생각해요!
타칸은 어떻게 생각하시나요?